src/controller/python/templates/python-CHIPClusters-cpp.zapt

```diff
@@ -90,6 +90,11 @@ void OnAttributeResponse<bool>(void * /* context */, bool value)
 {{#if isList}}
 static void On{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}ListAttributeResponse(void * context, {{zapTypeToDecodableClusterObjectType type ns=parent.name isArgument=true}} list)
 {
+#if 0
+    // TODO this code is not really used right now, doesn't work with nullable lists,
```

Comment:
You can delete these code directly instead of #if 0
This file will be removed in #12285

Reply:
Please do not add TODO in tempalte code: we have a todo bot that will add a  LOT of comments and issues in github for generated code. Use this: {{! ... }} for comments.

Unfortunately ./.github/workflows/lint.yml did  not catch this because it only checks zzz_generated

Reply:
Ah I see, didn't realize that. I will use the zap comment syntax, and I will just remove the code instead of #if 0. Thanks.

Reply:
Done.